generator/ServiceClientGeneratorLib/Generators/TestFiles/SmokeTestsV2.tt

```diff
@@ -8,6 +8,7 @@
 	AddLicenseHeader();
 #>
 using System;
+using System.Collections.Generic;
 using System.Threading.Tasks;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
(I'm not reviewing the PR, but...) Is there a reason why you're using MSTest as the test framework? 

The [hand-written files](https://github.com/aws/aws-sdk-net/blob/main/sdk/test/Services/SecretsManager/SmokeTests/SmokeTest.cs) we have right now use xUnit (meaning they can run in parallel) and using xUnit will allow you to test your PR without making any changes to our build system (as long as they're found by the projects in this folder: https://github.com/aws/aws-sdk-net/tree/v4-development/sdk/test/SmokeTests).